src/lib/components/Map/map.tsx

```diff
@@ -34,7 +34,16 @@ export const Map: React.FC<MapPropsType> = (props) => {
         width: sceneSize.width,
         height: sceneSize.height,
     });
-    const { scale, setNewScale } = useZoom({ ref: mapRef });
+    const [minScale, setMinScale] = React.useState(0.5);
+    const { scale, setNewScale } = useZoom({ ref: mapRef, minScale: minScale });
+
+    React.useEffect(() => {
+        const minScale = Math.min(
+            0.3,
```

Comment:
Minor thought:

When testing, I felt `0.3` might give a bit too small minimum zoom? Tested with e.g. `0.45` and minimum zoom was not as small.

Otherwise, resizing window etc. gave nice handling of minimum zoom!